connectors/salesforce/src/main/resources/cards/text.properties

```diff
@@ -9,9 +9,13 @@ senderinfo.name=Name
 senderinfo.account=Account
 senderinfo.phone=Phone Number
 
-senderinfo.opportunity.title=Opportunity
-senderinfo.opportunity.role=Role
-senderinfo.opportunity.probability=Probability
-senderinfo.opportunity.amount=Amount
+opportunity.header=[Salesforce] {0}
```

Comment:
Please add a locale test and these properties to the text_xx.properties (I usually go with UPPERCASE since it's still readable by everyone on the team w/o requiring google translate, but it looks like salesforce has a mix of german and spanish in the tests).

Reply:
That is a good idea with UPPERCASE in xx files.
Updated text_xx.properties.

Reply:
@RyanBard FYI, the XX values were just supposed to be gibberish.